Xamarin.Forms.Xaml/XamlLoader.cs

```diff
@@ -93,7 +93,7 @@ public static void Load(object view, string xaml, Assembly rootAssembly, bool us
 					var rootnode = new RuntimeRootNode(new XmlType(reader.NamespaceURI, reader.Name, null), view, (IXmlNamespaceResolver)reader) { LineNumber = ((IXmlLineInfo)reader).LineNumber, LinePosition = ((IXmlLineInfo)reader).LinePosition };
 					if (XamlFilePathAttribute.GetFilePathForObject(view) is string path) {
 						VisualDiagnostics.RegisterSourceInfo(view, new Uri($"{path};assembly={view.GetType().GetTypeInfo().Assembly.GetName().Name}", UriKind.Relative), ((IXmlLineInfo)rootnode).LineNumber, ((IXmlLineInfo)rootnode).LinePosition);
-						VisualDiagnostics.SendVisualTreeChanged(null, view);
+						VisualDiagnostics.SendVisualTreeChanged(view, null);
```

Comment:
I don't think this change is required

Reply:
Reverted: We're not using this event and talking offline, I can see your point. It makes sense to leave it for now.